aws/index.js

```diff
@@ -0,0 +1,14 @@
+const nodemailer = require('nodemailer');
+const AWS = require('aws-sdk');
+
+AWS.config.update({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  region: process.env.AWS_REGION,
+});
+
+export const transporter = nodemailer.createTransport({
+  SES: new AWS.SES({
+    apiVersion: '2010-12-01',
+  }),
```

Comment:
Won't this create a new object on every import? Is that intentional?

Reply:
I followed their documentation so I'm not exactly sure https://nodemailer.com/transports/ses/

Reply:
Are you saying it should be without the `new`?

Reply:
Hm, never mind. I don't think it will create a new one. My bad. This is fine.